www/includes/easyparliament/templates/html/section/wrans_index.php

```diff
@@ -0,0 +1,7 @@
+<div class="full-page__row">
+
+    <?php $content = $wrans; $title = "Written Answers"; include '_business_section.php'; ?>
+    <?php $search_title = 'Search Written Answers'; include '_search.php'; ?>
```

Comment:
This template is also being used for written-answers-and-statements, in which case it's searching both, if I understand it right. Perhaps this could be made clearer in general in this template?